src/lib/mongoBackend/mongoRegisterContext.cpp

```diff
@@ -47,17 +47,22 @@ using namespace mongo;
 */
 HttpStatusCode mongoRegisterContext
 (
-  RegisterContextRequest*   requestP,
-  RegisterContextResponse*  responseP,
-  const std::string&        tenant,
-  const std::string&        servicePath
+  RegisterContextRequest*              requestP,
+  RegisterContextResponse*             responseP,
+  std::map<std::string, std::string>&  uriParam,
+  const std::string&                   tenant,
+  const std::string&                   servicePath
+
 )
-{    
+{
+    // FIXME P10: we have been discussing about changing the "notifyFormat" name for something more neutral and more
```

Comment:
As you can see, by the moment I'm using the existing "nofityFormat". However, this could be fixed in the next PR (yours) which finalize the feature in the upper layers.

NTC